### DISCUSSION TITLE
[Bug] Window Store Apps Missing Icon in Taskbar

### DISCUSSION THREAD

**Discussion by yodaluca23** [valinet/ExplorerPatcher#1426] [General]
So I've had this issue for over half a year, where apps from the Microsoft Windows Store would appear blank on the taskbar, and I could never figure out what caused it. But as I was installing ExploererPatcher on another computer, I noticed it happening there too, so I have come to find, that this is a bug of ExplorerPatcher. I have tried all the recommended troubleshooting steps, including clearing the icon cache. 

ScreenShots:
![image](https://user-images.githubusercontent.com/67206487/236638783-25f674e0-de7b-491a-9d10-f0b76307defb.png)

**Comment by pyrates999** [comment]
I think this is this: https://github.com/valinet/ExplorerPatcher/issues/1108
